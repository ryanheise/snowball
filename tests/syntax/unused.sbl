// Test unused routines and groupings.
externals (stem)
integers (x)
booleans (b)
routines (r)
groupings (g)
define g 'aeiou'
define r as (
   unset b
   try (next set b)
   $x = size
   b
   $(x > 2)
   g
)

define stem as true
