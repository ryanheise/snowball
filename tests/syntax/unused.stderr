syntax/unused.sbl:8: warning: routine 'r' defined but not used
